### ISSUE TITLE
Save State Location

### ISSUE THREAD

**Issue by Dantasstic** [retrogamecorps/s30-add-on-pack#7]
Currently the SRM and the STATE files are saved in the ROM directory for each game, and show up on the game list menu in the GUI because of this. 

Can these be set to go to the correct folders for each type of save instead?

**Comment by deusxmach1na** [comment]
I double checked and mine all go to the /saves folder you had to create on the root of the sd card.  These are the settings you need in `retroarch.cfg` (make sure you check the other configs like `retroarch-dc.cfg`, `retroarch-gb.cfg`, etc.):
```
savefile_directory = "/mnt/extsd/saves"
savefiles_in_content_dir = "false"
savestate_auto_index = "true"
savestate_auto_load = "false"
savestate_auto_save = "false"
savestate_directory = "/mnt/extsd/saves"
savestate_thumbnail_enable = "false"
savestates_in_content_dir = "false"
```

**Comment by retrogamecorps** [comment]
They should all be going to the "saves" folder as of v1.5 of the add-on pack, if you have an older version with older cfg files they were saving in the roms folder.

**Comment by Dantasstic** [comment]
Why are there "saves" and "states" folders in the retro folder of this pack if they are not used at all?

**Comment by Dantasstic** [comment]
> 
> 
> They should all be going to the "saves" folder as of v1.5 of the add-on pack, if you have an older version with older cfg files they were saving in the roms folder.

Ok, the main issue is you make no mention of creating a folder called "saves" in the root during your video guide, which is what I followed. It is only mentioned in your written guide, but anyone following along with the video is gonna end up with this issue of saves appearing in their game list.

**Comment by retrogamecorps** [comment]
I've removed the saves/states folders from the retro folder, they are a remnant of a test I was conducting to see if separating them would make them persist (they didn't).  Thanks for the catch.

I've added wording to the video description to inform viewers to be sure to read the written guide as it is constantly updated.  I realize not a lot of people read the description but hopefully that'll at least nudge them in the right direction.
